install.sh

```diff
@@ -306,7 +306,7 @@ nvm_do_install() {
     command printf "${SOURCE_STR}"
   else
     case "${NVM_PROFILE-}" in
-      ".bashrc" | ".bash_profile" | ".zshrc")
+      *"/.bashrc" | *"/.bash_profile" | *"/.zshrc")
```

Comment:
It's already in `nvm_do_install` - if that's not tested or is hard to test, then splitting it into a separate function is fine.